src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -9698,4 +9704,17 @@ public static Value expected_cold_medicine_cabinet(ScriptRuntime controller) {
 
     return value;
   }
+
+  public static Value absorbed_monsters(ScriptRuntime controller) {
+    AggregateType type =
+        new AggregateType(DataTypes.MONSTER_TYPE, GreyYouManager.absorbedMonsters.size());
```

Comment:
You made this an array rather than a set.
Why?

As an array:

foreach index, mon in xxx

As a set:

foreach mon in XXX

And

XXX contains mon

Reply:
By set, I assume you mean Map? Changes has been made.

Reply:
Yes.

Array:  monster []
Sat: boolean [monster]